meta-resin-common/recipes-containers/docker/docker/docker.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=network.target docker.socket var-lib-docker.mount
+Requires=docker.socket var-lib-docker.mount
+
+[Service]
+Type=notify
+Restart=always
+ExecStart=/usr/bin/docker daemon -s btrfs
```

Comment:
If we're using systemd socket activation, which looks like we do since this service requires `docker.socket`, the daemon needs to be started with `-H fd://`